src/meta/processors/admin/AdminClient.cpp

```diff
@@ -36,6 +36,10 @@ folly::Future<Status> AdminClient::transLeader(GraphSpaceID spaceId,
     if (it == peers.end()) {
         return Status::PartNotFound();
     }
+    if (peers.size() == 1 && peers.front() == leader) {
```

Comment:
make sure the leader is being selected?

Reply:
No, transfer leader will failed in some case when replica is 1. see https://github.com/vesoft-inc/nebula-storage/issues/458.